connector_dns/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
you need to adapt the template accordingly... (check the previous one for the content)